otredis_test.go

```diff
@@ -21,98 +24,155 @@ func init() {
 
 func TestMain(m *testing.M) {
 	// in-memory redis
-	s, err := miniredis.Run()
+	miniRedis, err := miniredis.Run()
 	if err != nil {
 		panic(err)
 	}
-	defer s.Close()
+	defer miniRedis.Close()
+
+	redisAddr = miniRedis.Addr()
+
 	client = redis.NewClient(&redis.Options{
-		Addr: s.Addr(),
+		Addr: redisAddr,
 	})
 
 	os.Exit(m.Run())
 }
 
+// SET
+
 func TestSet(t *testing.T) {
 	ctx := context.Background()
 
-	span, ctx := opentracing.StartSpanFromContext(ctx, "test-params")
+	span, ctx := opentracing.StartSpanFromContext(ctx, "test-set")
 	ctxClient := WrapRedisClient(ctx, client)
 	callSet(t, ctxClient, "with span")
 	span.Finish()
 
 	spans := tracer.FinishedSpans()
-	if len(spans) != 2 {
-		t.Fatalf("should be 2 finished spans but there are %d: %v", len(spans), spans)
-	}
+	assert.Equal(t, 2, len(spans), "the number of finished spans is invalid")
```

Comment:
testify/assert has convenient `New` method which allows to skip passing `t` everywhere:
Also `assert` package contains `Len` method. So this can be rewritten as
```go
func TestSet(t *testing.T) {
    assert := assert.New(t)
    ....
    assert.Len(spans, 2, "the number of finished spans is invalid")
``

Reply:
Yep, good point